lib/conform/translate.ex

```diff
@@ -133,7 +133,11 @@ defmodule Conform.Translate do
                      nil -> mapping.default
                      var when is_binary(var) ->
                        case System.get_env(var) do
-                         nil -> mapping.default
+                         nil ->
+                           case mapping.default do
+                             nil -> raise missing_env_var(var, key)
```

Comment:
I don't think we should raise here - at this point we haven't checked the .conf to see if there is a value provided for this mapping, and raising here would imply that if `env_var: "FOO"` is part of the mapping, then `"FOO"` must always be present in the environment or configuration will always fail. I think we only want to fail if `"FOO"` isn't available, and there is neither a default value or a value provided in the .conf for the mapping - this makes `env_var` useful as both the sole source of configuration (just leave default nil and provide nothing in the .conf), as well as a fallback source.